src/DotNetWorker.Grpc/Definition/GrpcFunctionDefinition.cs

```diff
@@ -7,24 +7,26 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using Microsoft.Azure.Functions.Worker.Converters;
 using Microsoft.Azure.Functions.Worker.Grpc.Messages;
 using Microsoft.Azure.Functions.Worker.Invocation;
-using Microsoft.Azure.Functions.Worker.Converters;
 
 namespace Microsoft.Azure.Functions.Worker.Definition
 {
     internal class GrpcFunctionDefinition : FunctionDefinition
     {
+        private const string FunctionsApplicationDirectoryKey = "FUNCTIONS_APPLICATION_DIRECTORY";
+
         public GrpcFunctionDefinition(FunctionLoadRequest loadRequest, IMethodInfoLocator methodInfoLocator)
         {
             EntryPoint = loadRequest.Metadata.EntryPoint;
             Name = loadRequest.Metadata.Name;
             Id = loadRequest.FunctionId;
 
-            string? scriptRoot = Environment.GetEnvironmentVariable("FUNCTIONS_WORKER_DIRECTORY");
+            string? scriptRoot = Environment.GetEnvironmentVariable(FunctionsApplicationDirectoryKey);
```

Comment:
Should we check both for timebeing (until host changes to set the new env variable is fully rolled out). That allows us to merge and release this version without waiting for host changes?

`Environment.GetEnvironmentVariable("FUNCTIONS_APPLICATION_DIRECTORY")
                ?? Environment.GetEnvironmentVariable("FUNCTIONS_WORKER_DIRECTORY");`

Reply:
Yeah, good point -- that would be a good approach. Then this will just start working once the host supports it.